Improve instructions about setting GMT_LIBRARY_PATH env variable (#2136)
GenericMappingTools/pygmt: doc/install.rst

This can happen if you have multiple versions of GMT installed.

You can tell PyGMT exactly where to look for ``libgmt`` by setting the
``GMT_LIBRARY_PATH`` environment variable to the directory where ``libgmt.so``,

> ``libgmt.dylib`` or ``gmt.dll`` can be found for Linux, macOS and Windows,

Reviewer: ```suggestion
``libgmt.dylib`` or ``gmt.dll`` can be found on Linux, macOS or Windows,
```

respectively.

For Linux/macOS, add the following line to your shell configuration file

> (usually `~/.bashrc` for Bash on Linux and `~/.zshrc` for Zsh on macOS)::

Author: ```suggestion
(usually ``~/.bashrc`` for Bash on Linux and ``~/.zshrc`` for Zsh on macOS)::
```

> (usually `~/.bashrc` for Bash on Linux and `~/.zshrc` for Zsh on macOS)::

Reviewer: Should we use the tab thing for Linux/macOS/Windows instructions? Similar to what is currently used for conda/mamba?

Author: I don't think it's necessary.

    export GMT_LIBRARY_PATH=$HOME/anaconda3/envs/pygmt/lib

> For Windows, adding the ``GMT_LIBRARY_PATH`` environment variable following the

Reviewer: ```suggestion
For Windows, add the ``GMT_LIBRARY_PATH`` environment variable following these
```

You can tell PyGMT exactly where to look for ``libgmt`` by setting the
``GMT_LIBRARY_PATH`` environment variable to the directory where ``libgmt.so``,
``libgmt.dylib`` or ``gmt.dll`` can be found on Linux, macOS or Windows,
respectively.

> For Linux/macOS, adding the following line to your shell configuration file

Author: ```suggestion
For Linux/macOS, add the following line to your shell configuration file
```
